src/test/cypress/integration/quiz-exercise/QuizExerciseAssessment.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { artemis } from '../../support/ArtemisTesting';
+import shortAnswerQuizTemplate from '../../fixtures/quiz_exercise_fixtures/shortAnswerQuiz_template.json';
+import multipleChoiceQuizTemplate from '../../fixtures/quiz_exercise_fixtures/multipleChoiceQuiz_template.json';
+
+// Accounts
+const admin = artemis.users.getAdmin();
+const tutor = artemis.users.getTutor();
+const student = artemis.users.getStudentOne();
+
+// Requests
+const courseManagementRequest = artemis.requests.courseManagement;
+
+// Common primitives
+let course: any;
+let quizExercise: any;
+
+describe('Quiz Exercise Assessment', () => {
+    before('Set up course', () => {
+        cy.login(admin);
+        courseManagementRequest.createCourse().then((response) => {
+            course = response.body;
+            courseManagementRequest.addStudentToCourse(course.id, student.username);
+            courseManagementRequest.addTutorToCourse(course, tutor);
+        });
+    });
+
+    afterEach('Delete Quiz', () => {
+        deleteQuiz();
+    });
+
+    after('Delete Course', () => {
+        cy.login(admin);
+        courseManagementRequest.deleteCourse(course.id);
+    });
+
+    describe('MC Quiz assessment', () => {
+        before('Creates a quiz and a submission', () => {
+            createQuiz();
+        });
+
+        it('Assesses a mc quiz submission automatically', () => {
+            cy.login(student);
+            courseManagementRequest.startExerciseParticipation(course.id, quizExercise.id);
+            courseManagementRequest.createMultipleChoiceSubmission(quizExercise, [0, 2]);
+            cy.wait(5000);
+            cy.visit('/courses/' + course.id + '/exercises/' + quizExercise.id);
+            cy.contains(quizExercise.title);
+            cy.contains('Score');
```

Comment:
I think this assertion is too weak. Basically you go through all the effort of creating a quiz exercise and participating in it as a student to only verify that at the automatic assessment shows the exercise title and that the string "Score" is shown. But there is no assertion on what the result of the assessment actually is. So we should at least test that the student got a certain score to make sure that the automatic assessment properly assesses the submission

Reply:
I agree with this so i made the assertions a little more interesting.